File: cos/cos.java
public class cos {
    // ATOMIC_FUNCTION:  cos(val)
    // TAGS: math，numeric
    // LANG:  java
    // FUNCTION: cos(val)



    public static void main(String args[]) {
        double start_val = 0;
        double expected_end_val = 1;

        double end_val = Math.cos(start_val);
        

        // TEST
        if(end_val == expected_end_val) {
            System.out.println("PASS");
        } else {
            System.out.println("FAIL");
        }
    }
}

File: cos/cos.php
<?php
// ATOMIC_FUNCTION:  cos($val)
// TAGS: math，numeric
// LANG:  php
// FUNCTION: cos($val)

$start_val = 0;
$expected_end_val = 1;

$end_val = cos($start_val);

// TEST
if($end_val == $expected_end_val) {
	echo "PASS";
} else {
	
	echo "FAIL";
}

?>
